docs/src/modules/components/MarkdownElement.js

```diff
@@ -81,7 +81,7 @@ const Root = styled('div')(({ theme }) => ({
   '& h3': {
     ...theme.typography.h6,
     margin: '20px 0 10px',
-    fontFamily: ['"PlusJakartaSans-Bold"'].join(','),
+    fontFamily: ['"PlusJakartaSans-ExtraBold"'].join(','),
```

Comment:
@oliviertassinari Any idea on how we can use the bold weight? It hurts the heading hierarchy a bit having the h3 styling be extra-bold too.

Reply:
I think that we can ask @siriwatknp. He seems to have been iterating multiple times on the best way to load fonts.

For context, I noticed it loading the page in BrowserStack in a Windows 10 machine, where this font-weight is not installed by default. The header had the default user-agent font, I believe.

Reply:
will add the bold weight.